<map id="ChimeraTK::Device::setException" name="ChimeraTK::Device::setException">
<area shape="rect" id="node2" href="$class_chimera_t_k_1_1_unified_backend_test.html#a459185defdc5648adc1a123aa3164f07" title="Test repeated setException() has no effect (in particular, no additional exceptions in async transfer..." alt="" coords="208,5,412,47"/>
<area shape="rect" id="node5" href="$class_chimera_t_k_1_1_unified_backend_test.html#a5da51d23a58cd487bc88b157638c7c58" title="Test setException() disables asynchronous read transfers. " alt="" coords="208,71,412,112"/>
<area shape="rect" id="node6" href="$class_chimera_t_k_1_1_unified_backend_test.html#a06378a6b5f452bec5b7d8c3406bccb46" title="Test exactly one runtime_error in the _readQueue per async read accessor. " alt="" coords="208,136,412,177"/>
<area shape="rect" id="node7" href="$class_chimera_t_k_1_1_unified_backend_test.html#ad4fa09feca6336b169d490c976e64570" title="Test doReadTransferSynchonously throws runtime_error after setException() until recovery. " alt="" coords="208,201,412,243"/>
<area shape="rect" id="node8" href="$class_chimera_t_k_1_1_unified_backend_test.html#a4466ae0346858234f1c645f30517587b" title="Test write operations throw after setException() " alt="" coords="208,267,412,308"/>
<area shape="rect" id="node3" href="$class_chimera_t_k_1_1_unified_backend_test.html#a7434d5256809a1b2cf634d39f32a83df" title="Execute all tests. " alt="" coords="460,136,664,177"/>
<area shape="rect" id="node4" href="$test_numeric_addressed_backend_unified_8cpp.html#a9f5533e72ee880d6d818844f51879211" title="BOOST_AUTO_TEST_CASE" alt="" coords="712,143,903,170"/>
</map>
